### PR TITLE
Compatibility with MOLGENIS 8.3

### PR DESCRIPTION
**Pull request by tommydeboer**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/system tested
- [x] User documentation updated
- [x] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: mcmd/commands/add.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def add_user(args):
 @command
 def add_group(args):
     io.start('Adding group %s' % highlight(args.name))
-    post(api.group(), data={'name': args.name.lower(), 'label': args.name})
+    post(api.group(), data={'name': args.name, 'label': args.name})
```
Comment (fdlk) [inline mcmd/commands/add.py]: Have you considered making this dependent on version as well?

File: mcmd/commands/make.py [hunk 2]
```diff
@@ -97,15 +117,38 @@ def _get_user_group_membership(user: str, group_name: str) -> Optional[Membershi
 
 
 def _get_memberships_for_user(user_name: str) -> List[Membership]:
-    # Don't use a query here because the indexer might not've finished yet
```
Comment (fdlk) [inline mcmd/commands/make.py]: This sounds ominous

File: mcmd/commands/make.py [hunk 3]
```diff
@@ -51,39 +68,42 @@ def make(args):
...
 
-    if not matches:
-        raise McmdError('No group found for role %s' % upper_snake(role))
+    role = response.json()['items'][0]
 
-    return matches[max(matches, key=int)]
+    if 'group' not in role:
+        raise McmdError('Role {} is not a group role'.format(role_name))
+
+    return response.json()['items'][0]['group']['name']
```
Comment (fdlk) [inline mcmd/commands/make.py]: or `role['group']['name']`

**Comment by fdlk** [comment]
The add.gif still uppercases the role:
Schermafbeelding 2020-02-04 om 11 10 31

File: mcmd/commands/make.py (hunk 2 above)
Comment (tommydeboer) [reply mcmd/commands/make.py]: Yes it does, doesn't it 😅 

I remember writing this after I had a timing issue with a script like:
```
add group test
make henk test_manager
```

Now I just wasn't sure if that made sense so I tried it a couple of times and couldn't reproduce it.

Comment (tommydeboer) [reply mcmd/commands/make.py]: In an ideal world each command would validate its own success, I think. Working around timing issues by avoiding queries didn't feel right to me, so I removed it.

**Pull request merged**